nautobot/dcim/templates/dcim/rackgroup.html

```diff
@@ -7,10 +7,10 @@
             {% clone_button object %}
         {% endif %}
         {% if perms.dcim.change_rackgroup %}
-            {% edit_button object key="pk" %}
+            {% edit_button object %}
         {% endif %}
         {% if perms.dcim.delete_rackgroup %}
-            {% delete_button object key="pk" %}
+            {% delete_button object %}
         {% endif %}
```

Comment:
Should be able to remove the `buttons` block from this template entirely now.

Reply:
Removing the block buttons removed the `clone button`. However these objects do not `clone_fields` to begin with so I do not think it was supposed to be there.